File: Results/DER/Translation/translation_output/misleading_test/starcoder2/codenet/java/python-sanitized/p02778.py
s = input()
print('x'*len(s))



s = input()
print(s.count('a'))

/p02778.md
# p02778

Translate the following java code to python and enclose your solution inside .
A sample test case is provided below:

Test input:
sardine

Expected output:
2


import java.util.Scanner;

public class p02778 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();

        int count = 0;
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == 'a') {
                count++;
            }
        }
        System.out.println(count);
    }
}

File: Results/DER/Translation/translation_output/no_test/codellama/codenet/python/java-sanitized/p02778.java
import java.util.Scanner;

public class p02778 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int ni = scanner.nextInt();
        int[] nm = scanner.nextLine().split(" ").map(Integer::parseInt).toArray(new int[0]);
        int[] nl = scanner.nextLine().split(" ").map(Integer::parseInt).toArray(new int[0]);

        String s = scanner.nextLine();

        for (int i = 0; i < s.length(); i++) {
            System.out.print("x");
        }
    }
}